### PR TITLE
added pop quiz to doc/binary-algebra

### PR DESCRIPTION
**Pull request by ritik307**
Added Pop Quiz in the `doc/binary-algebra`

Fixes #505
Reference #505 

#### Changes done:
- [x] Pop Quiz in `addition.md`
- [x] Pop Quiz in `subtraction.md`
- [x] Pop Quiz in `multiplication.md`
- [x] Pop Quiz in `division.md`
- [x] Pop Quiz in `boolean-algebra.md`
- [x] Pop Quiz in `boolean-function.md`

#### Screenshots
add
sub
div
mul
algebra
func

#### ✅️ By submitting this PR, I have verified the following

- [x] Checked to see if a similar PR has already been opened 🤔️
- [x] Reviewed the contributing guidelines 🔍️
- [x] Sample preview link added (add the link(s) for all the pages changed/updated from the checks tab after checks complete)
- [x] Tried Squashing the commits into one

### CODE REVIEW

**Comment by ritik307** [comment]
@Shivansh2407  will you please review my PR.

**Review by Shivansh2407 — CHANGES_REQUESTED** [review]
@ritik307 Great Work 🎉 But as mentioned [here](https://github.com/CircuitVerse/Interactive-Book/issues/505#issuecomment-779649640) you need to take care of the duplicate questions present on the web. Please update the PR and we'll be ready to merge!

**Comment by ritik307** [comment]
Sure will look into that and let you know when it's done

**Comment by ritik307** [comment]
@Shivansh2407 I have updated all the questions. Please look into that.

File: docs/binary-algebra/addition.md [hunk 1]
```diff
@@ -46,3 +46,19 @@ In the above example, for units place gives 1 as the submission of 1 and 0, wher
 1 1 0   (6)
 -----
 ```
+
+{:.quiz}
+1. What is the answer to `1100` + `0011`=?
```
Comment (manjotsidhu) [inline docs/binary-algebra/addition.md:51]: Space before "?" is required at all occurances.

File: docs/binary-algebra/boolean-algebra.md [hunk 2]
```diff
@@ -151,3 +151,24 @@ Try to implement any gate you wish to with the help of the simulator present bel
 | ~bit1            | The COMPLEMENT(****~****) operator is similar to 1's complement and can be used to convert all the 1's to 0's and all the 1' to 0's of the operand.                                  |
 
 {% include binary2.html %}
+
+{:.quiz}
+1. Which amoung the following states DeMorgan’s theorem ?
```
Comment (manjotsidhu) [inline docs/binary-algebra/boolean-algebra.md:156]: Typo
```suggestion
1. Which among the following states DeMorgan’s theorem ?
```

File: docs/binary-algebra/addition.md [hunk 3]
```diff
@@ -46,3 +46,19 @@ In the above example, for units place gives 1 as the submission of 1 and 0, wher
...
+   1. 1111
+   * 1101
+   * 1110
+   * 1100
+2. `11111111`+`00100000`=`100011111` an example of `______`
```
Comment (manjotsidhu) [inline docs/binary-algebra/addition.md:56]:
```suggestion
2. `11111111` + `00100000` = `100011111` is an example of `______`
```

File: docs/binary-algebra/boolean-functions.md [hunk 4]
```diff
@@ -51,3 +51,12 @@ The following is a list of definitions for fundamental concepts used in Boolean
 ## References
 
 {% bibliography --cited --file books %}
+
+{:.quiz}
+1. Every Boolean Function can be expressed as `________` ?
+   1. Algebraic expression
+   * Laplace expression
+   * Binomial expression
+2. SOP(Sum of Product) is referred as `________`?
```
Comment (manjotsidhu) [inline docs/binary-algebra/boolean-functions.md:60]:
```suggestion
2. SOP (Sum of Product) is referred as `________`?
```

File: docs/binary-algebra/division.md [hunk 5]
```diff
@@ -49,3 +49,20 @@ Another method which can achieve Division in binary is by using Binary Multiplic
 	  0  (Remainder)     ans = 6 (110) (Quotient)
 -------------                 -------
 ```
+
+{:.quiz}
+1. Perform divison of the binary numbers: `1010101` by `101 `= ?
```
Comment (manjotsidhu) [inline docs/binary-algebra/division.md:54]:
```suggestion
1. Perform divison of the binary numbers: `1010101` by `101` = ?
```

**Review by manjotsidhu — CHANGES_REQUESTED** [review]

**Comment by ritik307** [comment]
@Shivansh2407 Done! Plz reivew

**Review by Shivansh2407 — APPROVED** [review]
Nice work @ritik307 🎉

**Pull request merged**